src/workloads/transactions/LLTInsertHighLtc.yml

```diff
@@ -0,0 +1,282 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  Workload to Benchmark the effect of LongLivedTransactions on an Insert workload.
+
+GlobalDefaults:
+  # Start: The following block gets updated for the other configurations.
+  # In-memory: Database size works out about 12GB.
+  InitialDocumentCount: &InitialNumDocs 10000000
```

Comment:
This file was generated and the InitialDocumentCount, SecondaryDocumentCount, GlobalRateValue and ThreadsValue were changed.